test_generate_commands.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from generate_commands import get_starting_line
+import ast
+
+class TestGetStartingLine(unittest.TestCase):
+
+    def test_empty_script(self):
+        script = ""
+        tree = ast.parse(script)
+        self.assertEqual(get_starting_line(tree), 1)
+
+    def test_only_definitions(self):
+        script = """
+def foo():
+    pass
+
+class Bar:
+    def method(self):
+        pass
+"""
+        tree = ast.parse(script)
+        self.assertEqual(get_starting_line(tree), 1)  # Now it should return 1
```

Comment:
@coderabbitai Thats because some testcases actually have starting lines, which is when get_starting_lines shouldnt return 1 and should return the line number where the starting line is. Or am I seeing this wrong? You tell me.